engine/runtime-compiler/src/main/scala/org/enso/compiler/phase/ImportResolver.scala

```diff
@@ -179,7 +179,7 @@ class ImportResolver(compiler: Compiler) {
             u.compilationStage(CompilationStage.INITIAL)
           }
         )
-        compiler.ensureParsed(mod)
+        compiler.ensureParsed(mod, false)
```

Comment:
Don't use caches to [avoid this NPE](https://github.com/enso-org/enso/actions/runs/7810331000/job/21303561703?pr=8924#step:8:1591).